modules/library/src/test/scala/zio/elasticsearch/ElasticQuerySpec.scala

```diff
@@ -1165,6 +1165,17 @@ object ElasticQuerySpec extends ZIOSpecDefault {
             )
           )
         },
+        test("regexp") {
+          val query                      = regexp("stringField", "t.*st")
+          val queryTs                    = regexp(TestDocument.stringField, "t.*st")
+          val queryWithCaseInsensitive   = regexp("stringField", "t.*st").caseInsensitiveTrue
+          val queryTsWithCaseInsensitive = regexp(TestDocument.stringField, "t.*st").caseInsensitiveTrue
```

Comment:
Rename this one to `queryWithCaseInsensitive`.

Reply:
And add another one val `queryWithSuffix`, where you can do something like this: `TestDocument.stringField.raw`.